chain/index/chain-indexing-overview-for-rpc-providers.MD

```diff
@@ -0,0 +1,258 @@
+# ChainIndexer Documentation for RPC Providers
+
+# Table of Contents
+
```

Comment:
@BigLep Added TOC.

Reply:
Thanks.  Although I'm realizing we don't really need this if this goes in Lotus docs since it renders a toc on the right hand side: 
https://lotus.filecoin.io/lotus/developers/local-network/
<img width="247" alt="image" src="https://github.com/user-attachments/assets/d8cb3c00-d83e-41a3-af3e-611367012c12">

Anyways, I guess we can leave for now until it moves over but we don't need to worry about finessing it (other than making sure our overall document structure is good).

Reply:
Removed this as we don't need it.